content/en/docs/instrumentation/php/manual.md

```diff
@@ -46,25 +44,52 @@ using the `Sdk::builder()` method. The returned `SdkBuilder` instance gets the
 providers related to the signals, tracing and metrics, in order to build the
 `OpenTelemetry` instance.
 
-You can build the providers by using the `TracerProvider::builder()` and
-`MeterProvider::builder()` methods. It is also strongly recommended to define a
-`Resource` instance as a representation of the entity producing the telemetry;
-in particular the `service.name` attribute is the most important piece of
-telemetry source-identifying info.
+You can build the providers by using the `TracerProvider::builder()`,
+`LoggerProvider::builder()`, and `MeterProvider::builder()` methods. It is also
+strongly recommended to define a `Resource` instance as a representation of the
+entity producing the telemetry; in particular the `service.name` attribute is
+the most important piece of telemetry source-identifying info.
 
 ### Example
 
 ```php
 <?php
-$resource = ResourceInfoFactory::defaultResource();
+
+use OpenTelemetry\API\Common\Signal\Signals;
+use OpenTelemetry\API\Trace\Propagation\TraceContextPropagator;
+use OpenTelemetry\Contrib\Grpc\GrpcTransportFactory;
+use OpenTelemetry\Contrib\Otlp\MetricExporter;
+use OpenTelemetry\Contrib\Otlp\OtlpUtil;
+use OpenTelemetry\Contrib\Otlp\SpanExporter;
+use OpenTelemetry\SDK\Common\Attribute\Attributes;
+use OpenTelemetry\SDK\Common\Export\Http\PsrTransportFactory;
+use OpenTelemetry\SDK\Logs\LoggerProvider;
+use OpenTelemetry\SDK\Metrics\MeterProvider;
+use OpenTelemetry\SDK\Metrics\MetricReader\ExportingReader;
+use OpenTelemetry\SDK\Resource\ResourceInfo;
+use OpenTelemetry\SDK\Resource\ResourceInfoFactory;
+use OpenTelemetry\SDK\Sdk;
+use OpenTelemetry\SDK\Trace\Sampler\AlwaysOnSampler;
+use OpenTelemetry\SDK\Trace\Sampler\ParentBased;
+use OpenTelemetry\SDK\Trace\SpanProcessor\BatchSpanProcessorBuilder;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+use OpenTelemetry\SemConv\ResourceAttributes;
+
+require 'vendor/autoload.php';
+
+$resource = ResourceInfoFactory::defaultResource()->merge(ResourceInfo::create(Attributes::create([
+    ResourceAttributes::SERVICE_NAMESPACE => 'foo',
+    ResourceAttributes::SERVICE_NAME => 'bar',
+    ResourceAttributes::SERVICE_VERSION => '0.1',
+    ResourceAttributes::DEPLOYMENT_ENVIRONMENT => 'development',
+])));
 $transport = (new GrpcTransportFactory())->create('http://collector:4317' . OtlpUtil::method(Signals::TRACE));
```

Comment:
Updated the example to not use grpc (it's covered better in the exporters doc).

Reply:
Tried the [updated example](https://github.com/open-telemetry/opentelemetry.io/blob/2ad249e0de61477d94fe1832b48c00fa349aa6c9/content/en/docs/instrumentation/php/manual.md#example), now it can't find SpanExporter:
```
PHP Fatal error:  Uncaught Error: Class "OpenTelemetry\Contrib\Otlp\SpanExporter" not found in /Users/tnajaryan/work/experiments/otel-php/GettingStarted.php:32
Stack trace:
#0 {main}
  thrown in /Users/tnajaryan/work/experiments/otel-php/GettingStarted.php on line 32
```